source/patterns/@aws-solutions-constructs/aws-eventbridge-lambda/README.md

```diff
@@ -61,12 +61,15 @@ _Parameters_
 |:-------------|:----------------|-----------------|
 |existingLambdaObj?|[`lambda.Function`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda.Function.html)|Existing instance of Lambda Function object, providing both this and `lambdaFunctionProps` will cause an error.|
 |lambdaFunctionProps?|[`lambda.FunctionProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-lambda.FunctionProps.html)|User provided props to override the default props for the Lambda function.|
+|existingEventBus?|[`events.IEventBus`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-events.IEventBus.html)| Optinally user-provided custom EventBus, uses `default` EventBus if this property is not set. Providing both this and `eventBusProps` results an error.|
```

Comment:
Let's go with the following for lines 63 and 64:

|existingEventBusInterface?|[`events.IEventBus`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-events.IEventBus.html)| Optional user-provided custom EventBus for construct to use. Providing both this and `eventBusProps` results an error.|
|eventBusProps?|[`events.EventBusProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-events.EventBusProps.html)|Optional user-provided properties to override the default properties when creating a custom Event Bus. Setting this value to {} will create a custom Event Bus using all default properties. If neither this nor existingEventBusInterface is provided the construct will use the default event bus. Providing both this and `existingEventBusInterface` results an error.|

Reply:
Note that we need to call this argument existingEventBusInterface to stress that it's the interface, not the construct (which end in Obj, like LambdaObj). There are a few constructs where we switched from Construct to Interface after release, so the name is wrong - but we're trying to stick with this pattern for new releases.

Reply:
taken care